wikipediapreview.php

```diff
@@ -60,7 +60,7 @@ function wikipediapreview_detect_deletion() {
 function wikipediapreview_guten_enqueue() {
 
 	// feature toggle of the gutenberg support, disabled by default
-	if ( ! $_GET['wikipediapreview_gutenburg'] ) {
+	if ( ! $_GET['wikipediapreview_gutenburg'] ) { // phpcs:ignore
```

Comment:
I could not get rid of the lint error, another search brings me to [get_query_var](https://developer.wordpress.org/reference/functions/get_query_var), but it doesn't work [per comment](https://developer.wordpress.org/reference/functions/get_query_var/#comment-4408)